bitnami/elasticsearch/templates/master/statefulset.yaml

```diff
@@ -391,6 +391,11 @@ spec:
           persistentVolumeClaim:
             claimName: {{ .Values.master.persistence.existingClaim }}
   {{- else }}
+  {{- if .Values.persistentVolumeClaimRetentionPolicy.enabled }}
+          persistentVolumeClaimRetentionPolicy:
+            whenDeleted: {{ .Values.persistentVolumeClaimRetentionPolicy.whenDeleted }}
+            whenScaled: {{ .Values.persistentVolumeClaimRetentionPolicy.whenScaled }}
+  {{- end }}
```

Comment:
According to [Kubernetes documentation](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#persistentvolumeclaim-retention) this settings are supposed to be located under `spec.persistentVolumeClaimRetentionPolicy`.

In this case, indent is wrong and these settings should be aligned with `volumeClaimTemplates`, not with `volumes`.